### DISCUSSION TITLE
I want native maps in the browser (and HTML)

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#108] [Wants]
Web maps and spatial data are the backbone of open data around the world. Governments promote spatial data infrastructure with the objective that everyone benefit equally. What has happened though is that huge companies have disproportionately leveraged this data to create centralized services that are available publicly (and for &quot;free&quot;), but at the cost of your privacy.

What we want is to make it easier to create and use web maps and spatial data by implementing maps in browsers. A native HTML `map` element could improve the map viewing experience on so many fronts, including privacy, accessibility, and progressive enhancement.

Our goal, and the goal of the [W3C Maps4HTML Community Group](https://www.w3.org/community/maps4html/) is to establish a standard method for web developers to include interactive web maps within HTML documents.

<hr>

https://webwewant.fyi/wants/61/

**Comment by ststimac** [comment]
Bocoup is conducting a survey on HTML maps https://twitter.com/mozhacks/status/1498588084925087745

**Comment by ghost** [comment]
@ststimac @aarongustafson Hi all! An idea exists the [open street map](https://www.openstreetmap.org/) project, could this request integrate the [open street map](https://www.openstreetmap.org/) in browsers?
